db/re/skill_tree.txt

```diff
@@ -3248,9 +3248,9 @@
 4068,2382,5,2422,1,0,0,0,0,0,0,0,0 //MI_ECHOSONG#Echo Song#
 4068,2383,5,2422,1,0,0,0,0,0,0,0,0 //MI_HARMONIZE#Harmonize#
 4068,2412,10,0,0,0,0,0,0,0,0,0,0 //WM_LESSON#Lesson#
-4068,2413,10,2417,1,0,0,0,0,0,0,0,0 //WM_METALICSOUND#Metallic Sound#
```

Comment:
1. Metalic sound max level reduced?
2. ID 2417 (Should be removed) as it was removed in the skill stree.

Reply:
2417 is removed in the next Improvement only, this is the 1st rebalance.

Reply:
This PR contains both part 1 and part 2 of the Minstrel/Wanderer patch.

Reply:
Then you have the wrong link for the 2nd rebalance ^^.
You have the Genetic Improvement link as 2nd.

Reply:
They are part of the same patch. If you click the image it will take you to the Minstrel/Wanderer part on the website: 
![image](https://user-images.githubusercontent.com/3521753/76857002-e8b03a80-682a-11ea-80b4-90a537cbd99f.png)

Reply:
Oh I thought that was the Homunculs part xD, my bad then lol.
Didn't check it completely. Since I can't translate from an image lol with Google Translator haha.